core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -285,6 +287,20 @@ trait WorkflowComponent {
       groupedSeq.map(_.toMap)
     }
 
+    def countWorkflowsByQueueStatusByUser: ReadAction[Map[String, Map[String, Int]]] = {
+      // Query for a Seq of (User, Workflow) tuples.
+      val userWorkflows = (for {
+        workflow <- findQueuedAndRunningWorkflows
+        submission <- submissionQuery if workflow.submissionId === submission.id
+        user <- rawlsUserQuery if submission.submitterId === user.userSubjectId
+      } yield (user, workflow)).result
```

Comment:
That's a good point.. it might be more efficient to do that in the database. Let me try that out next week.